libraries/aws/provisioning/test/unit/aws_iot_tests_provisioning_parser.c

```diff
@@ -225,14 +281,14 @@ static void _verifyParsedRejectedResponse( const AwsIotProvisioningRejectedRespo
     ( void ) pParsedData;
 
     /* Verify that the rejected response was parsed as expected. */
-    AwsIotProvisioning_Assert( pParsedData->errorCodeLength == pExpectedData->errorCodeLength );
-    AwsIotProvisioning_Assert( 0 == memcmp( pParsedData->pErrorCode,
-                                            pExpectedData->pErrorCode,
-                                            pExpectedData->errorCodeLength ) );
-    AwsIotProvisioning_Assert( pParsedData->errorMessageLength == pExpectedData->errorMessageLength );
-    AwsIotProvisioning_Assert( 0 == memcmp( pParsedData->pErrorMessage,
-                                            pExpectedData->pErrorMessage,
-                                            pExpectedData->errorMessageLength ) );
+    TEST_ASSERT( pParsedData->errorCodeLength == pExpectedData->errorCodeLength );
+    TEST_ASSERT( 0 == memcmp( pParsedData->pErrorCode,
```

Comment:
Unity provides TEST_ASSERT_EQUAL_MEMORY for this.

Reply:
Done